roles/qe-server/tasks/main.yml

```diff
@@ -16,6 +16,14 @@
     comment="USM QE test user account"
     state=present
 
+- name: Allow usmqe user to do anything via sudo
+  copy:
+    src=20_usmqe_sudoers
+    dest=/etc/sudoers.d/20_usmqe_sudoers
```

Comment:
No you can't and it's ok: the qe-server role is expected to be executed from remote machine (eg. one's laptop or jenkins slave) during qe server setup, remotely as a root. Is that what you are asking about?